client/reader/following-manage/index.jsx

```diff
@@ -86,6 +89,13 @@ class FollowingManage extends Component {
 		window.removeEventListener( 'resize', this.resizeListener );
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		const forceRefresh = nextProps.sitesQuery !== this.props.sitesQueryy;
```

Comment:
ugh, my editor started 🐛ing out and adding extra chars whenever I copy/pasted.  thanks for the 👁